src/ol/renderer/canvas/canvastilelayerrenderer.js

```diff
@@ -317,18 +317,28 @@ ol.renderer.canvas.TileLayer.prototype.prepareFrame =
   var tmpExtent = ol.extent.createEmpty();
   var tmpTileRange = new ol.TileRange(0, 0, 0, 0);
   var childTileRange, fullyLoaded, tile, tileState, x, y;
+  var drawableTile = (
+      /**
+       * @param {!ol.Tile} tile Tile.
+       * @return {boolean} Tile is selected.
+       */
+      function(tile) {
+        var tileState = tile.getState();
+        return tileState == ol.TileState.LOADED ||
+            tileState == ol.TileState.EMPTY ||
+            tileState == ol.TileState.ERROR && !useInterimTilesOnError;
+      });
```

Comment:
> For example, it might be nicer to be able to call source.getTile() and get the interim tile if the "main" one was not loaded (and yes, this would mean passing useInterimTilesOnError to getTile).

I thought about that. But `manageTilePyramid` is another [user](https://github.com/openlayers/ol3/blob/master/src/ol/renderer/layerrenderer.js#L320) of `getTile`.